cheroot/connections.py

```diff
@@ -155,14 +158,22 @@ def get_conn(self, server_socket):
 
             # Wait for the next tick to occur.
             return None
+        finally:
+            for fno in socket_dict:
```

Comment:
This causes `KeyError: '29 is not registered'` on the next line.
Maybe we need to do
```suggestion
            for fno in rlist:
```
WDYT?

Reply:
https://travis-ci.com/github/cherrypy/cheroot/jobs/362558590#L1622

Reply:
Resolving per https://github.com/cherrypy/cheroot/pull/301#issuecomment-660733161